### PR TITLE
Fix impedance control remappings

### PR DESCRIPTION
**Pull request by JosjaG**

### CODE REVIEW

**Comment by PetervDooren** [comment]
Was this also tested on the robot?

File: hsrb_impedance_control/launch/impedance_control_node.launch [hunk 1]
```diff
@@ -5,8 +5,13 @@
   <node pkg="hsrb_impedance_control"
         name="impedance_control"
         type="hsrb_impedance_control_node">
-    <remap from="/wrench_state" to="$(arg namespace)/wrist_wrench/compensated"/>
-    <remap from="~configuration_names" to="~config_names"/>
+    <remap from="~wrench_state" to="$(arg namespace)/wrist_wrench/compensated"/>
```
Comment (PetervDooren) [inline hsrb_impedance_control/launch/impedance_control_node.launch]: Why the '~' ? What does this do?

File: hsrb_impedance_control/launch/impedance_control_node.launch [hunk 2]
```diff
@@ -5,8 +5,13 @@
...
+    <!-- 
+    The following remap is included from the original launch file (hsrb_impedance_control package).
+    It is, however, unclear what this should do and is therefore commented.
```
Comment (PetervDooren) [inline hsrb_impedance_control/launch/impedance_control_node.launch]: Good that we add a comment for this but if we don't know what it does maybe its not smart to comment it

File: launch/hero_bringup.launch [hunk 3]
```diff
@@ -163,7 +160,7 @@
             <remap from="/hsrb/odom" to="$(arg namespace)/base/measurements" />
             <remap from="/hsrb/arm_trajectory_controller/command" to="$(arg namespace)/arm_trajectory_controller/command" />
             <remap from="/hsrb/omni_base_controller/command" to="$(arg namespace)/omni_base_controller/command" />
-            <include file="$(find hero_bringup)/hsrb_impedance_control/launch/impedance_control_node.launch" ns="$(arg namespace)" if="$(optenv ROBOT_REAL false)" >
+			<include file="$(find hero_bringup)/hsrb_impedance_control/launch/impedance_control_node.launch">
```
Comment (PetervDooren) [inline launch/hero_bringup.launch]:
```suggestion
            <include file="$(find hero_bringup)/hsrb_impedance_control/launch/impedance_control_node.launch">
```

Comment (PetervDooren) [reply launch/hero_bringup.launch]: use spaces for whitespace

File: hsrb_impedance_control/launch/impedance_control_node.launch (hunk 1 above)
Comment (MatthijsBurgh) [reply hsrb_impedance_control/launch/impedance_control_node.launch]: http://wiki.ros.org/Names#Resolving

Comment (PetervDooren) [reply hsrb_impedance_control/launch/impedance_control_node.launch]: I see, thanks

**Pull request merged**